src/Libraries/PythonNodeModels/PythonEvaluationHelper.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Linq;
+
+namespace PythonNodeModels
+{
+    /// <summary>
+    /// Enum of possible values of python engine versions.
+    /// TODO: Remove when dynamic loading logic is there then we no longer need a hard copy of the options
+    /// </summary>
+    public enum PythonEngineVersion
+    {
+        IronPython2,
+        CPython3
+    }
+
+    /// <summary>
+    /// Singleton class that other class can access and use for query loaded Python Engine info.
+    /// TODO: Make the Singleton class public when Dynamic loading is also ready.
+    /// </summary>
+    internal sealed class PythonEvaluationHelper
```

Comment:
How about naming this `PythonEngineSelector` or something like that?

Reply:
Will do